scenarios/pfcp-buffering.xml

```diff
@@ -35,7 +35,7 @@
                             starts="Pfcp-Buffer-Session-Modify-${#}"
                             ends="Pfcp-Buffer-Session-Modify-${#-1}">
                     <step name="Pfcp-Buffer-Session-Modify-${#}" requires="Check-Traffic-Buff-Drop-DL" delay="5"
-                          exec="mock-smf-cmd modify --session-count 5 --base ${#}0 --ue-pool 17.0.${#-1}.0/24 --enb-addr 140.0.10${#-1}.1"/>
+                          exec="${DOCKER_COMPOSE_CMD} exec -T mock-smf pfcpctl -c modify --count 5 --baseID ${#}0 --ue-pool 17.0.${#-1}.0/24 --nb-addr 140.0.10${#-1}.1"/>
```

Comment:
Sorry, I haven't noticed this before in pfcpsim reviews, but using `nb` might not be accurate enough. The Node B is the base station in the 3G system. I would use instead `gnb` since we use everywhere the 5G terminology.

Reply:
@osinstom, I agree with @daniele-moro. We changed `gnb` to `nb` because we thought it would be more generic but NodeB is indeed specific of UMTS

Reply:
that's a minor thing to me, but `nb` is more generic IMO - LTE has eNB, and 5G has gNB - they are all NodeBs :) Anyway, I'm fine with both.